### DISCUSSION TITLE
No such module 'WalletConnectPairing'

### DISCUSSION THREAD

**Discussion by lianghuiboy** [WalletConnect/walletconnect-monorepo#1427] [Developer Support]
<img width="1184" alt="image" src="https://user-images.githubusercontent.com/17612082/189473605-f50880bd-e1c3-4cc5-abca-1579987c9af6.png">

**Comment by chadyj** [comment]
Answered here https://github.com/WalletConnect/walletconnect-monorepo/discussions/1420#discussioncomment-3622847
